packages/squiggle-lang/src/rescript/Distributions/GenericDist/GenericDist_Types.res

```diff
@@ -8,6 +8,7 @@ type error =
   | NotYetImplemented
   | Unreachable
   | DistributionVerticalShiftIsInvalid
+  | ArgumentError(string)
```

Comment:
incorrect number of args or incorrect type of args?

Reply:
Sorry, we're gonna have trouble here. I (perhaps stupidly) piggybacked a refactor into #232 that got rid of `genericDist` and `error` from `GenericDist_Types` so that we're only working from `DistributionTypes.{error, genericDist}` (and `error` is now called `distributionError` as of one of the PRs that got merged while I was sleeping).